IdokladSdk/Clients/AccountClient.cs

```diff
@@ -32,5 +37,20 @@ public AccountClient(ApiContext apiContext)
         /// Gets users.
         /// </summary>
         public Users Users => _users ?? (_users = new Users(this));
+
+        /// <inheritdoc />
+        public ApiResult<bool> Delete(int id)
+        {
+            return Delete<bool>($"{ResourceUrl}/Users/" + id);
+        }
+
+        /// <summary>
+        /// Gives list of subscriptions.
+        /// </summary>
+        /// <returns>List of subscriptions.</returns>
+        public SubscriptionList SubscriptionList()
```

Comment:
Tady bych to možná udělal tak, že bych udělal třídu Subscriptions podobně jako je výš Users a Agendas. Uvnitř té třídy Subscriptions by pak byla jediná metoda List.
Možná Ti to kvůli jedné metodě teď bude připadat zbytečné, ale až pak budeme přidávat metodu pro získání aktuálního předplatného, tak pak tu metodu přidáš do té třídy Subscriptions.
Kdyžtak se třeba zeptej na názor Radka nebo Dejva.